main_logic/omni_realtime_client.py

```
@@ -33,6 +37,19 @@ class TurnDetectionMode(Enum):
+if not GEMINI_AVAILABLE and _GEMINI_IMPORT_ERROR is not None:
+    try:
+        diag_path = write_ssl_diagnostic(
+            event="gemini_sdk_import_failed",
+            output_dir=str(_config_manager.app_docs_dir / "logs" / "diagnostics"),
+            error=_GEMINI_IMPORT_ERROR,
+            extra={"stage": "module_import"},
+        )
+        if diag_path:
+            logger.warning(f"Gemini SDK import failed, diagnostic saved: {diag_path}")
+    except Exception:
+        pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)